### ISSUE TITLE
CUDNN_STATUS_BAD_PARAM ERROR

### ISSUE THREAD

**Issue by ghadahamed** [AlexeyAB/darknet#4012]
Dear @AlexeyAB,

Kindly can you help me in resolving this issue as I tried a lot and I couldn't solve it. I searched for the similar issues but no solution helped me.

I use this rep to train on my data using YOLO but V1 to follow a paper to share its results.
I did the same configurations required with YOLO V3 but with YOLO v1 arch.
When I train with the yolo-config of YOLO v3, everything gets fine while when I train with YOLO v1, I got the following error:
 cuDNN status Error in: file: ..\..\src\convolutional_layer.c : cudnn_convolutional_setup() : line: 234 : build time: Sep 22 2019 - 21:28:51
cuDNN Error: CUDNN_STATUS_BAD_PARAM
cuDNN Error: CUDNN_STATUS_BAD_PARAM

Kindly @AlexeyAB help me to resolve this?

Sorry for disturbance and thanks in advance

**Comment by ghadahamed** [comment]
This is my config file

[net]
# Testing
batch=64
subdivisions=1
# Training
# batch=64
# subdivisions=8
height=448
width=448
channels=3
momentum=0.9
decay=0.0005
saturation=1.5
exposure=1.5
hue=.1

learning_rate=0.001
policy=steps
steps=200,400,600,2000,3000
scales=2.5,2,2,.1,.1
max_batches = 4000
#1
[convolutional]
batch_normalize=1
filters=64
size=7
stride=2
pad=1
activation=leaky

[maxpool]
size=2
stride=2
#2
[convolutional]
batch_normalize=1
filters=192
size=3
stride=2
pad=1
activation=leaky

[maxpool]
size=2
stride=2
#3
[convolutional]
batch_normalize=1
filters=128
size=1
stride=2
pad=1
activation=leaky
#4
[convolutional]
batch_normalize=1
filters=256
size=3
stride=2
pad=1
activation=leaky
#5
[convolutional]
batch_normalize=1
filters=256
size=1
stride=2
pad=1
activation=leaky
#6
[convolutional]
batch_normalize=1
filters=512
size=3
stride=2
pad=1
activation=leaky

[maxpool]
size=2
stride=2
#7
[convolutional]
batch_normalize=1
filters=256
size=1
stride=2
pad=1
activation=leaky
#8
[convolutional]
batch_normalize=1
filters=256
size=1
stride=2
pad=1
activation=leaky
#9
[convolutional]
batch_normalize=1
filters=256
size=1
stride=2
pad=1
activation=leaky
#10
[convolutional]
batch_normalize=1
filters=256
size=1
stride=2
pad=1
activation=leaky
#11
[convolutional]
batch_normalize=1
filters=512
size=3
stride=2
pad=1
activation=leaky
#12
[convolutional]
batch_normalize=1
filters=512
size=3
stride=2
pad=1
activation=leaky
#13
[convolutional]
batch_normalize=1
filters=512
size=3
stride=2
pad=1
activation=leaky
#14
[convolutional]
batch_normalize=1
filters=512
size=3
stride=2
pad=1
activation=leaky
#15
[convolutional]
batch_normalize=1
filters=512
size=1
stride=2
pad=1
activation=leaky
#16
[convolutional]
batch_normalize=1
filters=1024
size=3
stride=2
pad=1
activation=leaky

[maxpool]
size=2
stride=2
#17
[convolutional]
batch_normalize=1
filters=512
size=1
stride=2
pad=1
activation=leaky
#18
[convolutional]
batch_normalize=1
filters=512
size=1
stride=2
pad=1
activation=leaky
#19
[convolutional]
batch_normalize=1
filters=1024
size=3
stride=2
pad=1
activation=leaky
#20
[convolutional]
batch_normalize=1
filters=1024
size=3
stride=2
pad=1
activation=leaky

#######
#21
[convolutional]
batch_normalize=1
size=3
stride=2
pad=1
filters=1024
activation=leaky
#22
[convolutional]
batch_normalize=1
size=3
stride=2
pad=1
filters=1024
activation=leaky
#23
[convolutional]
batch_normalize=1
size=3
stride=2
pad=1
filters=1024
activation=leaky
#24
[convolutional]
batch_normalize=1
size=3
stride=2
pad=1
filters=1024
activation=leaky

[local]
size=3
stride=2
pad=1
filters=256
activation=leaky

[dropout]
probability=.5

[connected]
output= 1715
activation=linear

[detection]
classes=2
coords=4
rescore=1
side=7
num=2
softmax=0
sqrt=1
jitter=.2

object_scale=1
noobject_scale=.5
class_scale=1
coord_scale=5

**Comment by spaul13** [comment]
@ghadahamed  did u able to solve the BAD PARAM problem? I am also getting the same problem for alexnet.cfg for the [connected] layer. Can u plz tell me how to get rid of this issue?

**Comment by ghadahamed** [comment]
@spaul13 Yes, v1 is not supported with CUDNN, so I set the CUDNN flag in the make file with 0 and this error is resolved.
